integration/fuse/bin/alluxio-fuse

```diff
@@ -88,17 +90,17 @@ umount_fuse() {
 
 fuse_stat() {
   local fuse_info=$("${JAVA_HOME}/bin/jps" | grep AlluxioFuse)
-  if [[ -z ${fuse_info} ]]; then    
+  if [[ -z ${fuse_info} ]]; then
     echo "AlluxioFuse: not running"
     return 1
   else
     echo -e "pid\tmount_point\talluxio_path"
     echo -e "$(ps aux | grep [A]lluxioFuse | awk -F' ' '{print $2 "\t" $(NF-2) "\t" $NF}')"
-    return 0    
+    return 0
   fi
 }
 
-USAGE_MSG="Usage:\n\t$0 [mount|umount|stat]"
```

Comment:
$0 represents current name of this script, even though alluxio-fuse may be renamed. I think there is no need to modify $0

Reply:
$0 will generate the full path, which is not common in the usage of a command

Reply:
@apc999 what do you think?

Reply:
basename $0 may be more appropriate

Reply:
@gpang I think  `basename $0` can be better